Polkadot status up to Decoded 2023 (#4998)
w3f/polkadot-wiki: docs/general/polkadot-v1.md

- **Security**, intended as how secure the blockspace is. In Proof-of-Stake (PoS) networks, this is
  directly related to how much stake is locked on validator nodes, how much variance in stake there
  is between validators (i.e. how easy it is to attack a single validator), and how many
  validators there are securing the network (i.e. how easy it is for colluding validators to exert
  influence on the network). Additionally, it is also important to look at how many validators are owned by single
  operators (this will determine the degree of social centralization of the network), and how many
  validators run on the same service provider (this will determine the degree of physical
  centralization of the network).
- **Flexibility**, intended as how flexible the blockspace is, what can be done with it, and what
  type of data can be stored. Data quality plays an important role depending on the type of network.
  One might avoid having situations in which poor quality data flood blockspace hindering the prompt
  execution of vital processes.
- **Availability**, intended as how available blockspace is and how difficult it is to access it. It

>   should not be too difficult to get your hands on it so that any business model (not only DeFi) can
>   thrive using it. Ideally, a marketplace must drive the blockspace price based on demand, with

Reviewer: ```suggestion
  should not be too difficult to get your hands on it so that any business model  can
  thrive using it. Ideally, a marketplace must drive the blockspace price based on demand, with
```

  secondary market options to ensure the usage of "second-hand" blockspace.

> Polkadot's has been designed around those core blockspace principles. However, its design can be

Reviewer: ```suggestion
Polkadot has been designed around those core blockspace principles. However, its design can be
```

> further improved. Tasks which are currently managed on the relay chain, such as balances transfers, staking, and governance, can be delegated to

Reviewer: ```suggestion
further improved such that the tasks which are currently managed on the relay chain, such as balances transfers, staking, and governance, can be delegated to
```

[system parachains](../learn/learn-system-chains.md) to increase flexibility and to focus the use of
the relay-chain to provide shared security and interoperability. Blockspace is only accessible
through slot auctions, but an auction winner has access to a "freighter of blocks" regardless it is
needed or not. This creates high entry barriers and it can lead to waste of energy and resources.

For more information about blockspace the [this interview](https://youtu.be/e1vISppPwe4) to Robert
Habermeier as well as [this article](https://www.rob.tech/polkadot-blockspace-over-blockchains/) by
him.

> ### A Perspective Shift: Upcoming Polkadot Feature Developments

Reviewer: ```suggestion
### A Perspective Shift: Upcoming Polkadot Features Development
```

---
id: polkadot-v1
title: Polkadot 1.0
sidebar_label: Polkadot 1.0
description: Polkadot's 1.0, Blockspace, Decentralization and Resilience.
keywords: [polkadot vison, polkadot 1.0, Blockspace, Resilience, Decentralization]
slug: ../polkadot-v1
---

Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
tens of thousands. Different projects have different value propositions, suggesting that the future
will be multi-chain and that inter-chain communication will be crucial to establish collaborations
and leveraging each other strengths.

## Polkadot 1.0

> Polkadot 1.0 reflects the status of Polkadot in 2023 at time of the release of the Polkadot runtime
> v1.0. This sections focuses on Polkadot 1.0 and some philosophical digressions about network

Reviewer: ```suggestion
Polkadot 1.0 reflects the status of Polkadot in 2023 at time of the release of the [Polkadot runtime
v1.0.0](https://github.com/paritytech/polkadot/releases/tag/v1.0.0). This sections focuses on Polkadot 1.0 and some philosophical digressions about network
```

---
id: polkadot-v1
title: Polkadot 1.0
sidebar_label: Polkadot 1.0
description: Polkadot's 1.0, Blockspace, Decentralization and Resilience.
keywords: [polkadot vison, polkadot 1.0, Blockspace, Resilience, Decentralization]
slug: ../polkadot-v1
---

> ## A Multi-chain Future

Reviewer 2: Is it possible we could combine the two sections, as Polkadot 1.0 is the Multi-chain future?  i.e.,

```
## Polkadot 1.0: A Multi-chain Future
```

Author: I do not fully agree with this as a multi-chain future can exist without polkadot. I mean, multi-chain is bigger than polkadot, polkadot just provides a way towards it. Does this makes sense?

Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
tens of thousands. Different projects have different value propositions, suggesting that the future
will be multi-chain and that inter-chain communication will be crucial to establish collaborations
and leveraging each other strengths.

## Polkadot 1.0

Polkadot 1.0 reflects the status of Polkadot in 2023 at time of the release of the Polkadot runtime
v1.0. This sections focuses on Polkadot 1.0 and some philosophical digressions about network
resilience and blockspace.

Polkadot is a Layer-0 blockchain that brings to the multi-chain vision the following innovations and
initiatives:

- **Application-specific Layer-1 (L1) blockchains (or parachains).** Polkadot is a sharded network

>   where transactions are processed parallelly within each shard. Polkadot shards can be heterogenous

Reviewer 2: ```suggestion
  where transactions are processed in parallel with each shard. Polkadot shards can be heterogenous
```

  - [Substrate Builders Program](./builders-program.md)
  - [Polkadot Developer Heroes Program](./dev-heroes.md)
  - [Edx Courses](https://www.edx.org/school/web3x)
  - Rust and Substrate Courses (coming soon)

### Polkadot's Representation

The Polkadot realy-chain has been represented as a relay chain surrounded by multiple parachains
attached to it. Parachains can communicate with each other using a generalized
[cross-chain messaging format (XCM)](./learn-xcm-index) allowing messages to be delivered across
chains having different logic.

### Polkadot's Additional Functionalities

The Polkadot relay-chain also manges [crowdloans](../learn/learn-crowdloans.md),
[auctions](../learn/learn-auction.md), [staking](../learn/learn-staking.md),
[accounts](./learn-accounts-index), [balances](../learn/learn-balance-transfers.md), and
[governance](../learn/learn-polkadot-opengov.md). Parachain slots or cores are leased in 6-month
chunks for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for
lease deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.

### Polkadot's Resilience

> Decentralization is a crucial aspect of blockchain networks, but there is a trade-off between:

Reviewer 2: I'm not sure if it would be good to mention the trilemma here? (decentralization vs. scalability vs. security)

Author: Yep, I can briefly mention it (although I think the trilemma is over-cited by so many projects that it is not so interesting anymore IMO)

Reviewer 2: Yeah true, that's why I wasn't sure if it was even interesting to even address 😅

  energy to operate, and
- having a network that reaches consensus fast at the expense of being centralized, making it
  trivial to manipulate or attack.

Ideally, a network should be decentralized "enough" to make it practically impossible for someone to
exert manipulative or malicious influence on the network. So, **decentralization is a tool while the
goal is resilience**, which is achieved by additionally providing on-chain treasury and governance
mechanism allowing continuous incentives for the network's participants without relying on
intermediaries or centralized entities.

Currently, Polkadot 1.0 achieve resilience through the following strategies:

- Nominated Proof of Staking (NPoS) where the stake per validator is maximized and evenly
  distributed across validators.
- The [1KV program](./thousand-validators.md) aims to incentivize new operators to become network
  participants and further increase physical (how many validator nodes per service provider) and
  social decentralization (how many validator nodes per operator). Those can be explored with the
  [Polkawatch App](https://polkadot.polkawatch.app/).

> - An on-chain treasury and governance where every decision goes though public referenda and any

Reviewer 2: ```suggestion
- An on-chain treasury and governance (see: [OpenGov](../learn/learn-polkadot-opengov.md) where every decision goes though public referenda and any
```

  token holder can cast a vote.

### Polkadot's Blockspace

The design and realization of Polkadot 1.0 allowed its creators to truly realize the importance of
blockspace.

> A blockchain is a way to store data. The storage unit is the block, and once a block is finalized

Reviewer 2: ```suggestion
Blockspace represents the raw material that is produced as a result of the secure, trustless compute that blockchains provide.  A blockchain defines a way to propagate a set of state changes, where the storage unit for these state changes is the block, and once a block is finalized
```

Author: Blockspace can be non-secure and non-trustless. So I do not fully agree that blockspace "is produced as a result of secure and trustless computation". Do you also agree? I can produce a block with some data myself without a blockchain.

Reviewer 2: Of course, but if we're talking about Polkadot's specific goal, then Polkadot provides that.  I can edit it to reflect that particular point

onto the chain, it is practically impossible to modify the data within that block. In addition to
being tamper-proof, public permissionless blockchains like Polkadot store data that are visible to
everybody (i.e. public), and anybody can become a network participant (i.e. permissionless).

> So, the raw material produced by blockchains is blockspace; and how blockspace is organized,

Reviewer 2: ```suggestion
The raw material produced by blockchains is **blockspace**; and how blockspace is organized,
```

accessed, and used will eventually shape the degree of scalability of the network. There are three
distinctive blockspace features:

> - **security**, intended as how secure the blockspace is. In Proof-of-Stake (PoS) networks, this is

Reviewer 2: ```suggestion
- **Security**, intended as how secure the blockspace is. In Proof-of-Stake (PoS) networks, this is
```

  directly related to how much stake is locked on validator nodes, how much variance in stake there
  is between validators (i.e. how easy it is to spot and attack single validators), and how many
  validators there are securing the network (i.e. how easy it is to collude validators and stop the
  network). Additionally, it is also important to look at how many validators are owned by single
  operators (this will determine the degree of social centralization of the network), and how many
  validators run on the same service provider (this will determine the degree of physical
  centralization of the network).

> - **flexibility**, intended as how flexible the blockspace is, what can be done with it, and what

Reviewer 2: ```suggestion
- **Flexibility**, intended as how flexible the blockspace is, what can be done with it, and what
```

  type of data can be stored. Data quality plays an important role depending on the type of network.
  One might avoid having situations in which poor quality data flood blockspace hindering the prompt
  execution of vital processes.

> - **availability**, intended as how available blockspace is and how difficult it is to access it. It

Reviewer 2: ```suggestion
- **Availability**, intended as how available blockspace is and how difficult it is to access it. It
```

  should not be too difficult to get your hands on it so that any business model (not only DeFi) can
  thrive using it. Ideally, a marketplace must drive the blockspace price based on demand, with
  secondary market options to ensure the usage of "second-hand" blockspace.

Polkadot's has been designed around those core blockspace principles. However, its design can be

> further improved. Tasks such as balances transfers, staking, governance, etc. can be delegated to

Reviewer 2: ```suggestion
further improved. Tasks which are currently managed on the relay chain, such as balances transfers, staking, and governance, can be delegated to
```

[system parachains](../learn/learn-system-chains.md) to increase flexibility and to focus the use of
the relay-chain to provide shared security and interoperability. Blockspace is only accessible
through slot auctions, but an auction winner has access to a "freighter of blocks" regardless it is
needed or not. This creates high entry barriers and it can lead to waste of energy and resources.

For more information about blockspace the [this interview](https://youtu.be/e1vISppPwe4) to Robert
Habermeier as well as [this article](https://www.rob.tech/polkadot-blockspace-over-blockchains/) by
him.

> ### A Perspective Shift

Reviewer 2: ```suggestion
### A Perspective Shift: Polkadot 2.0
```

As with many other projects before Polkadot, at some point in time after achieving the
initially-planned goals, a perspective shift allows you to understand better what your project is
about and what you actually have built. This allows you to "run the extra mile" and achieve more
than what was originally planned.

The quote below by [Marcel Proust](https://en.wikipedia.org/wiki/Marcel_Proust) must remind us that
sometimes a perspective shift is crucial in _understanding_ the world, and perhaps it is more
important than _seeing more_ of the world.

_The only true voyage of discovery, the only fountain of Eternal Youth, would be not to visit
strange lands but to possess other eyes._

Thus, if we start to see Polkadot with _other eyes_ we can truly envision its potential and what it
could become.

Enter Polkadot 2.0, a long-term vision looking at Polkadot as a decentralized, secure, ubiquitous

> computing engine to power the next-generation of Web3 application.

Reviewer 2: A small excerpt here could be warranted, let me know what you think. Feel free to modify it, it is very rough.  I want to capture the concepts of "spaces", cores, etc: 

> In the context of Polkadot 2.0, the perspective shift will be crucial to realize the broader ideas it will bring due to its implementation.  From blockchains, a relatively two-dimensional structure, to defining governable, three-dimensional spaces of secure, decentralized computing.  This space would comprise multiple cores, similar to how a modern CPU is structured.  These cores can then communicate using a common language, XCM, which empowers applications built across these cores.   The goal is not a chain-centric future but an application-centric one that provides users true value and utility.
